### PR TITLE
Bootstrap5

### PR DESCRIPTION
**Pull request by davidacevedo**
Update react-gears to be bootstrap5 compatible with reactstrap. The changes also reflect the bootstrap 5 migration guide https://getbootstrap.com/docs/5.0/migration/

### CODE REVIEW

**Comment by davidacevedo** [comment]
[Update react-gears to Bootstrap 5](https://trello.com/c/wVWNQO3D/18-update-react-gears-to-bootstrap-5)

File: src/components/AddressInput.tsx [hunk 1]
```diff
@@ -177,11 +177,11 @@ const AddressInput = ({
           disabled={disabled}
         />
       </FormLabelGroup>
-      <Row className="no-gutters">
+      <Row className="gx-3">
```
Comment (aaronmars) [inline src/components/AddressInput.tsx:180]: `no-gutters` was renamed to `g-0` is 3 correct here?

File: src/components/DateInput.js [hunk 2]
```diff
@@ -270,7 +269,7 @@ export default class DateInput extends React.Component {
       dateFormat, defaultValue, keyboard, locale, onBlur, onChange, parse, positionFixed, value, state, ...props } = this.props; // eslint-disable-line no-shadow
     const { open } = this.state;
     const date = this.getCurrentDate();
-    const dropdownProps = open ? { positionFixed } : {};
+    const dropdownProps = open ? { strategy: positionFixed ? 'fixed' : undefined } : {};
```
Comment (aaronmars) [inline src/components/DateInput.js:272]: How about this instead?
```suggestion
    const dropdownProps = open && positionFixed ? { strategy: 'fixed' } : {};
```

**Review by aaronmars — CHANGES_REQUESTED** [review]

File: src/components/DateInput.js (hunk 2 above)
Comment (davidacevedo) [reply src/components/DateInput.js:272]: Yup, that looks better as well!

File: src/components/AddressInput.tsx (hunk 1 above)
Comment (davidacevedo) [reply src/components/AddressInput.tsx:180]: Seems like the gutters were not necessary. Will revert to g-0

**Review by aaronmars — APPROVED** [review]

File: src/components/AddressInput.tsx (hunk 1 above)
Comment (davidacevedo) [reply src/components/AddressInput.tsx]: Updated

File: src/components/DateInput.js (hunk 2 above)
Comment (davidacevedo) [reply src/components/DateInput.js]: Updated

**Review by emcpete — APPROVED** [review]

**Pull request merged**